build_tools/circle/build_doc.sh

```diff
@@ -130,7 +130,7 @@ else
     make_args=html
 fi
 
-make_args="SPHINXOPTS=-T $make_args"  # show full traceback on exception
+make_args="SPHINXOPTS=-j2 $make_args"
```

Comment:
The `-T` is already configured by default in the Makefile:

https://github.com/scikit-learn/scikit-learn/blob/effdd6e215c67f2ae8ed1e378ea1661e936059a4/doc/Makefile#L16-L17